test/test_private.py

```diff
@@ -127,7 +128,7 @@ def test_parse_duration(self):
             # different units
             ('10ns', datetime.timedelta(seconds=0.000_000_010)),
             ('11us', datetime.timedelta(seconds=0.000_011)),
-            ('12µs', datetime.timedelta(seconds=0.000_012)),  # U+00B5  # noqa: RUF001
+            ('12µs', datetime.timedelta(seconds=0.000_012)),  # U+00B5
```

Comment:
What's the reasoning for moving the pragma to a wider scope? Not saying we shouldn't do it, just wondering what the motivation is.

Reply:
This is because the "µs" part triggers a lint error. Since I already added RUF001 in `tool.ruff.lint.per-file-ignores`, let me see if I can remove this comment here.

Reply:
I did it for consistency, because we already have RUF001 in `tool.ruff.lint.per-file-ignores` for another file (`ops/_private/timeconv.py`) where there is also only one line that needs this.

Reply:
I think it's a couple of lines, but for what it's worth, my reasoning there (when I was doing the current ignores) was that `timeconv.py` is more tightly defined than `test_private` - ie. `timeconv` is only about time conversion and the micro/mu use is a core part of that, whereas `test_private` is more broadly testing any private utilities and it seemed riskier to say that any use of micro/mu should be ignored.

Reply:
Changing back since I don't have a preference here.